combat-trainer.lic

```diff
@@ -428,9 +428,11 @@ class LootProcess
       if @necro_heal && !game_state.necro_casting?
         game_state.wounds = check_health
         echo "Severity to Wounds: #{game_state.wounds}" if $debug_mode_ct
-        unless game_state.wounds.empty?
-          do_necro_ritual(mob_noun, 'consume', game_state)
-          return false
+        if @heal_threshold <= game_state.wounds.keys.max
+          unless game_state.wounds.empty?
```

Comment:
Is this conditional still necessary?

Reply:
I switched the logic around to have the unless check for nill before moving to the if > threshold